### PR TITLE
[FIX] Now the cylinder display as it should 🎉🎉🎉

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

File: source/render/render.c [hunk 1]
```diff
@@ -36,14 +39,15 @@ void	render(t_scene *scene, t_pixel_grid *pixel, t_ray *ray_pool)
...
-	while (hit_record)
+	cur_node = ray->hit_record_list;
+	while (cur_node)
 	{
-		// TODO: Need to handle front_face == false
-		ray->cache_color = compute_color(scene, hit_record->content);
-		hit_record = hit_record->next;
+		hit_record = cur_node->content;
+		ray->cache_color = compute_color(scene, hit_record);
+		cur_node = cur_node->next;
```
Comment (itislu) [inline source/render/render.c]: Why save the hit_record into a variable?

File: source/hit/rule_cylinder.c [hunk 2]
```diff
@@ -146,7 +132,7 @@ static void	setup_hit_record_side(
 	projection = vec3.ops->mul(
 			cylinder->norm, vec3.ops->dot(oc, cylinder->norm));
 	rec->norm = vec3.ops->sub(oc, projection);
-	rec->norm = vec3.ops->div(rec->norm, cylinder->d_param1);
+	rec->norm = vec3.ops->normalize(rec->norm);
```
Comment (itislu) [inline source/hit/rule_cylinder.c]: Unnecessary variable assignment

File: source/hit/hit_management.c [hunk 3]
```diff
@@ -24,3 +24,25 @@ bool	hit_object(t_ray *ray, t_obj *obj, t_hit_record *rec)
 		return (hit_cylinder(vec3, ray, obj, rec));
 	return (false);
 }
+
+double	min_possitive_t(double t1, double t2)
```
Comment (itislu) [inline source/hit/hit_management.c]: Typo: ~`min_possitive_t`~ `min_positive_t`

File: source/render/render.c (hunk 1 above)
Comment (LeaYeh) [reply source/render/render.c]: bc more clear

Comment (itislu) [reply source/render/render.c]: How about renaming `cur_node` to `hit_rec_node`?

**Review by itislu — APPROVED** [review]

**Pull request merged**